src/components/Header/_mocks_/index.js

```diff
@@ -100,6 +101,12 @@ const profile = {
     first_name: 'Sample',
     surname: 'User',
   },
+  description: (
```

Comment:
🔥

Reply:
Hey, does this mean the changes are approved?